src/VisualStudio/Core/Def/Implementation/ProjectSystem/Legacy/AbstractLegacyProject.cs

```diff
@@ -31,6 +31,8 @@ internal abstract partial class AbstractLegacyProject : ForegroundThreadAffiniti
         protected IProjectCodeModel ProjectCodeModel { get; set; }
         protected VisualStudioWorkspace Workspace { get; }
 
+        internal VisualStudioProject Test_VisualStudioProject => VisualStudioProject;
```

Comment:
```suggestion
        internal VisualStudioProject VisualStudioProject_TestOnly => VisualStudioProject;
```

Reply:
(this is just the naming convention we use everywhere else)

Reply:
I'll rename.